app/models/budget.rb

```diff
@@ -1,5 +1,5 @@
 class Budget < ActiveRecord::Base
-  belongs_to :user
+  belongs_to :user, optional: true
```

Comment:
I would not add this, because this allows for bad data.
And we want to have data integrity. We never want to have budgets that are not associated with any user.

Reply:
Done sha: 7807e6f